webvtt.html

```diff
@@ -1423,6 +1429,8 @@
       bidirectional formatting character in the text of the cue.</li>
       <li>A <a>WebVTT cue nbsp escape</a>, representing a U+00A0 NO-BREAK SPACE character in the
       text of the cue.</li>
+      <li>A <a>WebVTT cue numeric character reference</a>, representing a <a>Decimal numeric
```

Comment:
No need to capitalize these concepts in the prose, e.g. "entry settings object" is not capitalized.